sdk/arm/network/2020-03-01/armnetwork/example_firewallpolicies_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package armnetwork
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2019-05-01/armresources"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+
+const (
+	policyName    = "samplepolicy"
+	ruleGroupName = "sampleRuleGroup"
+)
+
+func getResourceGroupsOperations() armresources.ResourceGroupsOperations {
+	client, err := armresources.NewDefaultClient(getCredential(), nil)
+	if err != nil {
+		panic(err)
+	}
+	return client.ResourceGroupsOperations(subscriptionID)
+}
+
+func getFirewallPolicyRuleGroupsOperations() FirewallPolicyRuleGroupsOperations {
+	client, err := NewDefaultClient(getCredential(), nil)
+	if err != nil {
+		panic(err)
+	}
+	return client.FirewallPolicyRuleGroupsOperations(subscriptionID)
+}
+
+func getFirewallPoliciesOperations() FirewallPoliciesOperations {
+	client, err := NewDefaultClient(getCredential(), nil)
+	if err != nil {
+		panic(err)
+	}
+	return client.FirewallPoliciesOperations(subscriptionID)
+}
+
+func ExampleFirewallPolicyRuleGroupsOperations_BeginCreateOrUpdate() {
+	// create a new resource group to create resource in
+	rg := createResourceGroup(resourceGroupName, location)
```

Comment:
I'm kinda leaning towards omitting this to try and keep the example focused on just network.  I believe we omitted it for `armcompute`.  We can have a larger end-to-end sample that creates all the things elsewhere.  What do you think?

Reply:
Agreed. I'm thinking we can add one in the general ARM directory. We can talk about it later.